gohup_test.go

```diff
@@ -82,14 +82,12 @@ func Test_StartsAProcessInBackgroundAndLogOutput(t *testing.T) {
 
 	lp := &LocalProcess{}
 	pid, pidfile, err := lp.Start(o)
-	fmt.Println(pidfile)
-	//t.FailNow()
 
 	require.Eventuallyf(t, func() bool {
 		s, err := lp.QueryStatus(pidfile)
 		fmt.Println(s, err)
 		return s == StatusStopped
-	}, 30*time.Second, 1*time.Second, "process failed to stop")
```

Comment:
You can probably change the check interval to 10ms and remove the println as the quickest this test will run is currently 1s. This not really a huge deal.